src/pages/Payment.vue

```diff
@@ -77,15 +89,23 @@ export default {
         }
     },
     methods: {
+        showConfirmation: function() {
+            this.setMessage("Thanks, your payment request has been sent.", constants.MESSAGE_SUCCESS);
+            this.$router.message = this.message;
+            this.$router.push('/account');
+        },
         onRequestPayment() {
             Store.pendRecVouchers(
                 // on Success, route to /account
                 function(response) {
-                    this.$router.push('/account');
+                    this.showConfirmation();
                 }.bind(this),
                 // on Failure... hook for an alert?
                 function(error) {
-                }
+                    if (!Store.netMgr.online) {
```

Comment:
Ah, didn't realise it was only firing when offline - was just going by how it's done on other pages as far as I could tell. Will get rid of it and check I can get the error message to display.